antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -445,6 +453,12 @@ def clear_snapshot(self, variant_study: Study) -> None:
         self.invalidate_cache(variant_study, invalidate_self_snapshot=True)
         shutil.rmtree(self.get_study_path(variant_study), ignore_errors=True)
 
+        # Removes TS-generation related information from the database
```

Comment:
should go into `invalidate_cache`, in particular it needs to be applied to descendants